examples/hn.svelte.dev/src/routes/index.svelte

```diff
@@ -1,6 +1,7 @@
 <script context="module">
-	export function load({ page }) {
-		let host = page.host;
+	/** @type {import('@sveltejs/kit').Load} */
+	export function load({ url }) {
+		let host = url.host;
```

Comment:
This could probably be simplified to use `url.hostname`.

Reply:
good catch. i have a feeling this `url` thing will pay all sorts of dividends